fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/loader/impl/SnomedTermGraphLoader.java

```diff
@@ -0,0 +1,319 @@
+                                    JanusGraphManagement management = janusGraph.openManagement();
+                                    management.makeEdgeLabel(label).make();
+                                    management.commit();
+                                }
+
+                                Edge e = g.V(u).addE(label).to(v).next();
+
+                                if (!"0".equals(relationshipGroup)) {
+                                    g.E(e).property("group", relationshipGroup).next();
+                                }
+                            }
+                        }
+                    }
+
+                    if ((counter.get() % 10000) == 0) {
+                        log.info("counter: " + counter.get());
+                        g.tx().commit();
+                    }
+
+                    counter.getAndIncrement();
+                }
+            });
+
+            // commit any uncommitted work
+            g.tx().commit();
+        }
+    }
+
+    private static abstract class SnomedReleaseFileConsumer implements Consumer<String> {
+        private final List<String> lines = new ArrayList<>();
+        private String previousId = null;
+
+        @Override
+        public void accept(String line) {
+            if (collect(line)) {
+                lines.add(line);
+            } else {
+                processLines(Collections.unmodifiableList(lines));
+                lines.clear();
+                lines.add(line);
+            }
+        }
+
+        private boolean collect(String line) {
+            String[] fields = line.split("\\t");
+            String id = fields[0];
+            if (!id.equals(previousId)) {
+                previousId = id;
+                return lines.isEmpty();
+            }
+            return true;
+        }
+
+        private void processLines(List<String> lines) {
+            processLine(lines.get(lines.size() - 1));
+        }
+
+        // template method
+        public abstract void processLine(String line);
+    }
+
+    public static void main(String[] args) throws Exception {
+        Options options = null;
+        SnomedTermGraphLoader loader = null;
+        try {
+            long start = System.currentTimeMillis();
+
+            options = FHIRTermGraphLoader.Type.SNOMED.options();
+
+            CommandLineParser parser = new DefaultParser();
+            CommandLine commandLine = parser.parse(options, args);
+
+            loader = new SnomedTermGraphLoader(toMap(commandLine));
+            loader.load();
+
+            long end = System.currentTimeMillis();
+
+            log.info("Loading time (milliseconds): " + (end - start));
+        } catch (MissingOptionException e) {
+            System.out.println("MissingOptionException: " + e.getMessage());
+            HelpFormatter formatter = new HelpFormatter();
+            formatter.printHelp("SnomedTermGraphLoader", options);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
intentional?

Reply:
Since this is a main / command-line driver the e.printStackTrace() is there intentionally to assist in debugging.